src/Features/CSharp/Portable/ConvertToRawString/ConvertInterpolatedStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -178,7 +184,7 @@ protected override InterpolatedStringExpressionSyntax Convert(
         SyntaxFormattingOptions formattingOptions,
         CancellationToken cancellationToken)
     {
-        if (kind == ConvertToRawKind.SingleLine)
+        if (kind.HasFlag(ConvertToRawKind.SingleLine))
```

Comment:
>HasFlag [](http://example.com/codeflow?start=17&length=7)

Is this executed on the VS side? If so, will this method be called a lot. HasFlag is notoriously slow on net fx.

Reply:
It won't be called a lot. It's called only when you're on a raw string, and only a tiny number of times then.